modules/test/playwright/tests/blogs-web/blogEntry.spec.ts

```diff
@@ -150,3 +150,57 @@ test('LPD-24858 Categories with blank spaces in friendly URL', async ({
 			.join('/')}/${title}`
 	);
 });
+
+test('LPD-26753 The URL changes when a category is modified', async ({
+	apiHelpers,
+	blogsEditBlogEntryPage,
+	displayPageTemplatesPage,
+	page,
+	pageEditorPage,
+	site,
+}) => {
+	const vocabularyName = getRandomString();
+	const friendlyUrlCategories = ['category-1', 'category-2', 'category-3'];
+
+	const {categories} = await friendlyURLCategoriesSetup({
```

Comment:
could we rename it to blogsFriendlyUrlSetup or blogsCategorizedFriendlyUrlSetup

Reply:
We can go with `blogsCategorizedFriendlyUrlSetup`. 
I can apply the next PR, I'm working on top of this PR.